src/createAdder.js

```diff
@@ -22,7 +22,16 @@
  * @return {number}
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  let sum = initialValue;
+
+  return (...args) => {
+    if (args === undefined) {
+      return sum;
+    }
+    sum += args.reduce((suma, num) => suma + num, 0);
```

Comment:
return here
```suggestion
   return sum += args.reduce((suma, num) => suma + num, 0);
```
what is `suma`? don't name variable with translite

Reply:
i changed this variable name's